staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -113,10 +114,26 @@ type preparedAPIAggregator struct {
 	runnable runnable
 }
 
+// preparedToServeSignals holds signals/events that indicate this server has been fully initialized and is ready to serve requests
+type preparedToServeSignals struct {
```

Comment:
why a wrapper around another type? Just use `signal.Interface` directly

Reply:
I wanted to expose only the read-only interface

Reply:
But the current text exposes the full interface.
It looks to me like the reason for a struct here is to prepare for more signals in the future.  For example, one about RBAC being initialized.